diff --git a/package/kernel/mac80211/Makefile b/package/kernel/mac80211/Makefile
--- a/package/kernel/mac80211/Makefile
+++ b/package/kernel/mac80211/Makefile
@@ -123,6 +123,12 @@
 	help
 	  Enable runtime debugging information for mac80211
 
+config PACKAGE_OEM_WIFI_CREDS
+	bool "Include OEM WiFi Credential Generation"
+	default y
+	help
+	  Enables automatic generation of OEM-style WiFi credentials
+
 endmenu
 
 config MAC80211_RC_MINSTREL
